src/main/java/dev/arantes/lib/database/sql/Crud.java

```diff
@@ -10,37 +10,69 @@ public Crud(Connection connection, String table) {
         this.connection = connection;
         this.table = table;
     }
-    public void createTable(String format) throws SQLException {
-        PreparedStatement stm = connection.prepareStatement(
-                "CREATE TABLE IF NOT EXISTS " + table + " (" + format + ")"
-        );
-        stm.execute();
-        stm.close();
+
+    public void createTable(String format) {
+        PreparedStatement stm = null;
+
+        try {
+            stm = connection.prepareStatement(
+                    String.format("CREATE TABLE IF NOT EXISTS %s (%s)", table, format)
+            );
+
+            stm.execute();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        } finally {
+            finalizeStatement(stm);
+        }
     }
 
     public ResultSet executeQuery(String sql, Object... values) throws SQLException {
-        PreparedStatement stm = this.connection.prepareStatement(sql);
+        PreparedStatement stm = null;
+        ResultSet querySet = null;
 
-        for (int i = 0; i < values.length; i++) {
-            stm.setObject(i, values[i]);
+        try{
+            stm = connection.prepareStatement(sql);
+            for (int i = 0; i < values.length; i++) {
+                stm.setObject(i, values[i]);
+            }
+
+            querySet = stm.executeQuery();
+        } catch (SQLException e) {
+            e.printStackTrace();
+        } finally {
+            finalizeStatement(stm);
+
+            if (querySet != null) {
+                querySet.close();
```

Comment:
Closing the resultset before accessing it will prevent the user from pull the data out of it.